hack/oss-fuzz-build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eu
+# This script is used by OSS-Fuzz to build and run Limas fuzz tests continuously.
+# Limas OSS-Fuzz integration can be found here: https://github.com/google/oss-fuzz/tree/master/projects/lima
+# Modify https://github.com/google/oss-fuzz/blob/master/projects/lima/project.yaml for access management to Limas OSS-Fuzz crashes.
+printf "package store\nimport _ \"github.com/AdamKorcz/go-118-fuzz-build/testing\"\n" > "$SRC"/lima/pkg/store/register.go
```

Comment:
Thank you, but now `shfmt` is complaining about the extra space after the output redirection:

```suggestion
printf "package store\nimport _ \"github.com/AdamKorcz/go-118-fuzz-build/testing\"\n" >"$SRC"/lima/pkg/store/register.go
```